src/ui/activity-iframe-view-test.js

```diff
@@ -97,6 +106,30 @@ describes.realWin('ActivityIframeView', {}, (env) => {
       expect(activityIframePort.whenReady).to.have.been.calledOnce;
     });
 
+    it('should disallow scrolling within dialogs', async () => {
+      const iframe = new ActivityIframeView(
+        win,
+        activityPorts,
+        src,
+        activityArgs
+      ).getElement();
+
+      expect(iframe.scrolling).to.equal('no');
+    });
+
+    it('should (optionally) allow scrolling within dialogs', async () => {
```

Comment:
same here - this could be: "allows scrolling within dialogs behind an experiment"

Reply:
That's clearer 👍